src/pages/ScreenRouting/MediaScreenRouter.tsx

```diff
@@ -8,25 +8,30 @@ import useMedia from '#src/hooks/useMedia';
 import Loading from '#src/pages/Loading/Loading';
 import ErrorPage from '#components/ErrorPage/ErrorPage';
 import type { PlaylistItem } from '#types/playlist';
-import { isEpisode, isSeriesPlaceholder } from '#src/utils/media';
+import { isEpisode, isDeprecatedSeriesFlow } from '#src/utils/media';
 import MediaMovie from '#src/pages/ScreenRouting/mediaScreens/MediaMovie/MediaMovie';
 import MediaSeries from '#src/pages/ScreenRouting/mediaScreens/MediaSeries/MediaSeries';
 import MediaSeriesEpisode from '#src/pages/ScreenRouting/mediaScreens/MediaSeriesEpisode/MediaSeriesEpisode';
 import MediaLiveChannel from '#src/pages/ScreenRouting/mediaScreens/MediaLiveChannel/MediaLiveChannel';
 import { ScreenMap } from '#src/pages/ScreenRouting/ScreenMap';
+import { CONTENT_TYPE, SERIES_CONTENT_TYPE } from '#src/config';
 
 export const mediaScreenMap = new ScreenMap<PlaylistItem>();
 
-// register media screens
-mediaScreenMap.registerByContentType(MediaSeries, 'series');
-mediaScreenMap.registerByContentType(MediaSeriesEpisode, 'episode');
-mediaScreenMap.registerByContentType(MediaLiveChannel, 'livechannel');
-mediaScreenMap.registerByContentType(MediaStaticPage, 'page');
+// Register media screens
+
+// Register all series screens
+Object.keys(SERIES_CONTENT_TYPE).forEach((type) => {
```

Comment:
I would suggest we wait to implement support for multiple series content types. As this shows, in the absence of any concrete use cases we would simply map them all to the same screen, which begs the question as to why they would be considered different content types in the first place. My guess is that any content type that additionally has series attached to it would end up having different screens, tags, etc. Probably having the CONTENT_TYPE enum is a good enough consistency improvement for now.

Reply:
The problem I was also thinking about was showing 'series' tag on the Home page. We need to check the content type and this way of implementing it could solve both screen and tag problem. But I agree that we can use only `series` content type for now